src/ChristmasTree.java

```diff
@@ -0,0 +1,17 @@
+public class ChristmasTree {
+    public static void main(String[] args) {
+        for (int i = 0; i <= 5; i++) {
```

Comment:
Only 1 triangle. Please implement 2 more

Reply:
static void triangle2(int n) {
        for (int i = n; i >= 1; i--) {
            for (int j = 0; j < n - i; j++) {
                System.out.print(" ");
            }
            for (int j = 0; j < i; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }

    static void triangle3(int n) {
        for (int i = 1; i <= n; i++) {
            for (int j = 0; j < n - i; j++) {
                System.out.print(" ");
            }
            for (int j = 0; j < i; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }